fredshema/hoop-mobile-app
# feat: parking spots implementation

What does this PR do?
Retrieve parking spots from the database on appwrite

Description of Task to be completed?
Use appwrite Database and create a collection for the parking spots including attributes like title,address,time and price

How should this be manually tested?
Navigate to the Home screen and you will see available parking spots with all the datails like price per hour and the address and when you also click on one the category like car or van you will get the full category which is also data stored in the database on appwrite.

What are the relevant pivotal tracker/Trello stories?
https://trello.com/c/Xy5HjOW9/43


Questions:
Can you help me  on how you integrated the loading on notification screen cause I have to refresh the app and log in again to see changes I made on Appwrite on that page.
